test/Typewriter.Test/TestDataJava/com/microsoft/graph/requests/extensions/UserReferenceRequest.java

```diff
@@ -32,20 +32,27 @@
      * @param client         the service client
      * @param requestOptions the options for this request
      */
-    public UserReferenceRequest(@Nonnull final String requestUrl, @Nonnull final IBaseClient client, @Nullable final java.util.List<? extends com.microsoft.graph.options.Option> requestOptions) {
+    public UserReferenceRequest(String requestUrl, IBaseClient client, java.util.List<? extends com.microsoft.graph.options.Option> requestOptions) {
         super(requestUrl, client, requestOptions, User.class);
     }
 
+    public void delete(final ICallback<? super User> callback) {
```

Comment:
Can you please verify if the newly added methods  in this PR have summary?

Reply:
same thing, bad rebase.